src/osg/State.cpp

```diff
@@ -158,6 +222,19 @@ State::~State()
     // delete the GLExtensions object associated with this osg::State.
     if (_glExtensions)
     {
+        if (_transformStackID){
+            _glExtensions->glDeleteBuffers(1, &_transformStackID);
+        }
+       
+        // VRV_PATCH
+        // TDG: this was causing crashes in GLBufferObject because the extensions were getting deleted
+        // this line is not really usefull because we are using shared contexts so they all have a context ID of 0
+        // and use the same extensions.
+        //GLExtensions::Set(_contextID, 0);
+        // VRV_PATCH
+
+        // MAK REVIEW: The above was commented out in makOSG 3.4.  The below is new of 3.6.  Are they compatible?
```

Comment:
I can't think of any reason we should be deleting the GLExtensions (at least it is unclear that we can reasonably know when we should delete it) so lets get rid of the Get() and Set() code below.